tests/test_sql.py

```diff
@@ -19,14 +19,19 @@ def test_simple(tmp_library: TemporaryLibrary) -> None:
     assert len(folders) == 5
     assert len(glob.glob(tmp_library.libdir + "/**/*.pdf")) == 4
 
-    doc = papis.document.from_folder(os.path.join(tmp_library.libdir, "IH8J2JJP"))
+    doc = papis.document.from_folder(
+        os.path.join(
+            tmp_library.libdir,
+            "95e29aafd24e14274ab8a1bb7887e2b7-svard-magnus-and-no"
+            )
+        )
 
     info_name = os.path.join(os.path.dirname(__file__), "resources", "sql_out.yaml")
     with open(info_name, encoding="utf-8") as fd:
         data = yaml.load(fd, Loader=papis.yaml.Loader)  # type: ignore[attr-defined]
         expected_doc = papis.document.from_data(data)
 
-    assert expected_doc["ref"] == doc["ref"]
+    assert expected_doc["authors"] == doc["authors"]
```

Comment:
Why change the test? `authors` might be better, so just curious..

Reply:
Ah, fair enough. For the path, I think you'll have to add a 
```
papis.config.set("add-folder-name", "{doc[author]}")
```
to each test that needs that. I'm not sure why it works now, but the hash at the beginning of the folder name should be random and we want to avoid that.

Reply:
Ah, also, is the `doc[authors]` field correct? Papis uses `doc[author]` (singular), so we either need to fix that in the Zotero importer or fix this test.

Reply:
I think the assert worked because both fields were nonexistent :yum: